src/woeusb

```diff
@@ -2004,6 +2009,19 @@ util_check_function_parameters_quantity(){
 	return 0
 }; declare -fr util_check_function_parameters_quantity
 
+##Check every file in source directory for files bigger than max fat32 file size (~4GB)
+check_for_big_files(){
+	local source_fs_mountpoint="${1}"
+
+	for i in $(find ${source_fs_mountpoint}/ -type f); do
+		if (( $(du $i | cut -f -1) > 4194303 )); then
+			echo_with_color red "Error: File $i is larger than that supported by the fat32 filesystem. Use NTFS (--target-filesystem NTFS)."
+			exit 1
```

Comment:
Return a non-zero value instead of directly bailout from a function using `exit` so we can do more things in the caller function if we needed, for now, use the following snippet in `init` function:

```bash
if ! check_for_big_files\
	"${source_fs_mountpoint}"; then
	exit 1
fi
```